public/locales/en.json

```diff
@@ -185,6 +185,7 @@
         "untitledNote": "Untitled",
         "placeholder": "← Start by entering a title here\n===\nVisit the features page if you don't know what to do.\nHappy hacking :)",
         "invalidYaml": "The yaml-header is invalid. See <0></0> for more information.",
+        "deprecatedTags": "The comma-separated definition of tags in the YAML-metadata is deprecated. Convert them to a YAML-collection instead.",
```

Comment:
"Covert" sounds a bit more complicated as it is. Maybe just write "Use"?


Also:


Maybe we should add a faq entry? We did that for every deprecation.

Reply:
I think an FAQ entry is a good idea

Reply:
https://github.com/hedgedoc/hedgedoc.github.io/pull/43

Reply:
Done